eng/SourceBuild.props

```diff
@@ -0,0 +1,121 @@
+<Project>
+
+  <PropertyGroup>
+    <GitHubRepositoryName>runtime</GitHubRepositoryName>
```

Comment:
@ViktorHofer - I verified it's not feasible to remove this property redundancy.

Reply:
Seeing this diff, I remember that I filed an issue tracking it, but we closed it in triage: https://github.com/dotnet/source-build/issues/2029. More info about the risks/benefits in there.